File: GrocOrder/GrocOrder/ConsoleApp.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace GrocOrder
{
    class ConsoleApp
    {
        public void main()
        {
            CashRegister cr = new CashRegister();

            GrocItem gi = new GrocItem("Bread");
            cr.AddItem(gi);
        
            GrocItem gic = new GrocItem("Cereal");
            cr.AddItem(gic);
        
            GrocItem gif = new GrocItem("Coffee");
            cr.AddItem(gif);
        
            Console.WriteLine("In the Cash Register, there are " + cr.ItemCount + " items.");
        
        }
    }
}

File: eclipse/GrocOrder/src/demo/grocOrder/ConsoleApp.java
package demo.grocOrder;

public class ConsoleApp
{

    public static void main(String[] args)
    {
        // Create a hard-coded object
        PriceService ps = new PriceService();
        CashRegister cr = new CashRegister(ps);
        
        System.out.println("This is the hard-wired class dependency example...");

        GrocItem gi = new GrocItem("Bread");
        cr.AddItem(gi);
        
        GrocItem gic = new GrocItem("Cereal");
        cr.AddItem(gic);
        
        GrocItem gif = new GrocItem("Coffee");
        cr.AddItem(gif);
        
        System.out.println("In the Cash Register, there are " + cr.ItemCount() + " items.");
        
        System.out.println(cr.printFinalTotals());

    }

}
